docs/examples/psychopy/stimulus.md

```diff
@@ -0,0 +1,96 @@
+# Creating Stimuli
+
+This will be a guide on displaying different kinds of visual stimulus with PsychoPy.
```

Comment:
The following demonstrates how to create and time simple PsychoPy stimuli.

We are not doing an exhaustive tutorial here.